src/modules/fancyzones/editor/FancyZonesEditor/GridData.cs

```diff
@@ -120,25 +120,29 @@ public GridData(GridLayoutModel model)
             {
                 _colInfo.Add(new RowColInfo(model.ColumnPercents[col]));
             }
+
+            int maxIndex = 0;
+            for (int row = 0; row < _model.Rows; row++)
+            {
+                for (int col = 0; col < _model.Columns; col++)
+                {
+                    maxIndex = Math.Max(maxIndex, _model.CellChildMap[row, col]);
+                }
+            }
+
+            _zoneCount = maxIndex + 1;
```

Comment:
Before `ZoneCount` was equal to `maxIndex` not it is equal to `maxIndex+1`. Does it mean that indexing changed?

Reply:
No, indexing wasn't changed. I explained above the reason for it, `ZoneCount` was a misleading name.